cirq-core/cirq/transformers/dynamical_decoupling.py

```diff
@@ -177,7 +177,17 @@ def _calc_pulled_through(
     clifford_ops_in_moment: list[ops.Operation] = [
         op for op in moment.operations if _is_clifford_op(op)
     ]
-    return input_pauli_ops.after(clifford_ops_in_moment)
+    ps_after: ops.PauliString = ops.PauliString()
+    try:
```

Comment:
why? wouldn't it be cleaner to just convert to SingleQubitClifford?

Reply:
we need to pull through Paulis after multi-qubit cliffords as well.

Reply:
you can use the parent class `CliffordGate`
```py3
cirq.CliffordGate.from_op_list(clifford_ops_in_moment, [q for op in clifford_ops_in_moment for q in op.qubits])
```